Source/WebCore/platform/ValidationBubble.h

```diff
@@ -62,41 +60,36 @@ class ValidationBubble : public RefCounted<ValidationBubble> {
     };
 
 #if PLATFORM(GTK)
-    using ShouldNotifyFocusEventsCallback = Function<void(PlatformView*, bool shouldNotifyFocusEvents)>;
-    static Ref<ValidationBubble> create(PlatformView* view, String&& message, const Settings& settings, ShouldNotifyFocusEventsCallback&& callback)
-    {
-        return adoptRef(*new ValidationBubble(view, WTFMove(message), settings, WTFMove(callback)));
-    }
+    virtual ~ValidationBubble() = default;
```

Comment:
Let's add a warning comment to reduce risk of future error:

```suggestion
    // ValidationBubbleGtk is inherited from ValidationBubble.
    virtual ~ValidationBubble() = default;
```